VDF.GUI/Program.cs

```diff
@@ -35,7 +35,6 @@ public static AppBuilder BuildAvaloniaApp()
 			=> AppBuilder.Configure<App>()
 				.UsePlatformDetect()
 				.With(new X11PlatformOptions {  UseDBusFilePicker = false })
-				.With(new Win32PlatformOptions { UseWindowsUIComposition = true })
```

Comment:
I think the reason for having this was that without it, Acrylic style will lag on some machines when moving the window around. I cannot really find the commit on Avalonia where its removed and why and what they suggest to use instead. So I wonder if removing it entirely is the way to go.

Reply:
If I understand correctly, it was https://github.com/AvaloniaUI/Avalonia/pull/11898 that replaced this option by two new ones: [`CompositionMode`](https://github.com/AvaloniaUI/Avalonia/blob/11.0.4/src/Windows/Avalonia.Win32/Win32PlatformOptions.cs#L31-L70) and [`RenderingMode`](https://github.com/AvaloniaUI/Avalonia/blob/11.0.4/src/Windows/Avalonia.Win32/Win32PlatformOptions.cs#L10-L26).

`CompositionMode` is set to `Win32CompositionMode.WinUIComposition` [by default](https://github.com/AvaloniaUI/Avalonia/blob/11.0.4/src/Windows/Avalonia.Win32/Win32PlatformOptions.cs#L96-L108), and `RenderingMode` is set to `Win32RenderingMode.AngleEgl` [by default](https://github.com/AvaloniaUI/Avalonia/blob/11.0.4/src/Windows/Avalonia.Win32/Win32PlatformOptions.cs#L82-L94).

So I think we can keep this as-is?